content/updates/2020-06/2-option-screen-revamp.md

```diff
@@ -0,0 +1,15 @@
+# New: option screen revamp
+::: meta-data type="Improvement" features="Attribute Options, Data Governance" available="late June" link-to-doc="../articles/manage-simple-and-multi-selects-attribute-options.html"
+:::
+
+We improved the `Option screen` user experience to ease creating or editing attribute options, as the single and multi-select attributes are among the attributes you use the most.
```

Comment:
```suggestion
We have improved the `Attribute option screen` to make it easier to create or edit attribute options, since single and multi-select attributes are among the attributes you use the most.
```